history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc

```diff
@@ -235,6 +250,10 @@ But being earnestly desirous to obtain the liberty of their imprisoned friends,
 the friends of London renewed their solicitations to the king for their liberation,
 and at length obtained the following warrant to Sir Robert Sawyer, attorney-general:
 
+[.embedded-content-document.legal]
```

Comment:
Another weird letter....ugh!

Reply:
yeah, this is like that other one. i don't know what to do with it.  if there wasn't already a `sss` below, we could move it down. For lack of a better idea, let's go with what you did here. It looks about like the original did. 👍